.github/workflows/scripts/test/test_check_python_deps.py

```
@@ -0,0 +1,131 @@
+        """Should ignore comments and blank lines in requirements."""
+        with patch("importlib.metadata.distributions", return_value=[]):
+            with patch.object(
+                check_python_deps,
+                "REQUIREMENTS_FILE",
+                unittest.mock.MagicMock(
+                    exists=lambda: True,
+                    read_text=lambda: "# comment\n\npackage==1.0\n",
+                ),
+            ):
+                result = check_python_deps.get_missing_dependencies()
+                self.assertEqual(result, ["package"])
```

Ruff (0.14.10)

Use a regular `assert` instead of unittest-style `assertEqual`

Replace `assertEqual(...)` with `assert ...`

(PT009)